sound/soc/sof/sof-priv.h

```diff
@@ -344,7 +344,8 @@ struct snd_sof_dev {
 	bool msi_enabled;
 
 	/* client devices */
-	struct sof_mfd_client *sof_audio;
+	struct sof_mfd_client *sof_ssp_audio;
+	struct sof_mfd_client *sof_hdmi_audio;
 
```

Comment:
@ranj063 snd_sof_dev should mostly be a platform independent structure so, sof_ssp_audio and sof_hdmi_audio shouldn't be here. We can add here an array or a list of sof_mfd_clients and then each platform will populate the list with the desired clients.

Reply:
done